redirects.ts

```diff
@@ -149,7 +149,7 @@ function generateDocsRedirects(basePath: string): RedirectRule[] {
 		},
 
 		// SQL Guide
-		{ from: withBase('/sql-guide'), to: withBase('/developers/sql-guide/') },
+		{ from: withBase('/sql-guide'), to: withBase('/') },
 
```

Comment:
We also need to create redirects for all these pages being deleted (to redirect to `/` or else where) so that these deleted pages do not become dead links.